cypress/e2e/regression/add_owner.cy.js

```diff
@@ -2,11 +2,19 @@ import * as constants from '../../support/constants'
 import * as main from '../../e2e/pages/main.page'
 import * as owner from '../pages/owners.pages'
 import * as addressBook from '../pages/address_book.page'
+import * as ls from '../../support/localstorage_data.js'
 
 describe('Add Owners tests', () => {
   beforeEach(() => {
-    cy.visit(constants.setupUrl + constants.SEPOLIA_TEST_SAFE_1)
     cy.clearLocalStorage()
+    cy.visit(constants.setupUrl + constants.SEPOLIA_TEST_SAFE_1, {
+      onBeforeLoad: (win) => {
+        win.localStorage.setItem(
+          constants.localStorageKeys.SAFE_v2__dismissPushNotifications,
+          JSON.stringify(ls.dismissedNotifications),
+        )
+      },
+    })
```

Comment:
Do we have to add this and similar changes to our e2e tests because the banner is now always visible for owned safes and we want to hide it there?

Reply:
yeah this is just because the banner is not showing up for some of the tests and it blocks elements that it's trying to click. So if we end up getting rid of the banner we can get rid of this also